auth-tokens/src/main/java/com/palantir/tokens/auth/UnverifiedJsonWebToken.java

```diff
@@ -85,6 +88,14 @@ public abstract class UnverifiedJsonWebToken {
     @Value.Parameter
     public abstract Optional<String> getUnverifiedOrganizationId();
 
+    /**
+     * Returns the unverified expiration time, i.e. the "exp" claim, of the JWT
+     * or absent if the JWT does not contain the "exp" claim.
+     */
+    @Safe
+    @Value.Parameter
+    public abstract Optional<OffsetDateTime> getUnverifiedExpirationTime();
```

Comment:
`OffsetDateTime` is not the correct type to use here. This is just a offset-less timestamp so it should be `Instant`.

```suggestion
    public abstract Optional<Instant> getUnverifiedExpirationTime();
```